pkg/objectfile/object_file.go

```diff
@@ -191,15 +193,6 @@ func (f *ObjectFile) computeBase(addr uint64) error {
 	return nil
 }
 
-// mapping stores the parameters of a runtime mapping that are needed to
-// identify the ELF segment associated with a mapping.
-type mapping struct {
```

Comment:
Ah, I think this changed because I put all structs at the beginning of the file to make refactoring easier and then forgot to put it back 🙈 
I will put it back with a comment if you want